docs/src/pages/components/app-bar/BackToTop.tsx

```diff
@@ -57,9 +45,13 @@ function ScrollTop(props: Props) {
 
   return (
     <Zoom in={trigger}>
-      <div onClick={handleClick} role="presentation" className={classes.root}>
+      <Box
+        onClick={handleClick}
+        role="presentation"
+        sx={{ position: 'fixed', bottom: 2, right: 2 }}
```

Comment:
There is no spacing unit on these properties

```suggestion
        sx={{ position: 'fixed', bottom: 16, right: 16 }}
```